godot-codegen/src/special_cases/special_cases.rs

```diff
@@ -108,6 +108,13 @@ pub fn is_godot_type_deleted(godot_ty: &str) -> bool {
         return true;
     }
 
+    // UniformSetCacheRD has been reintroduced in 4.3
+    // see: https://github.com/godotengine/godot/commit/5a988456
+    #[cfg(since_api = "4.3")]
+    if godot_ty == "UniformSetCacheRD" {
+        return true;
+    }
```

Comment:
Is the `#[cfg(since_api = "4.3")]` necessary? If it's absent from all versions, we can just take it into the lower list, inside `match` (including comment).

Reply:
**Massive** brainfart on my side – it should return "false" (this type is **NOT** deleted since 4.3, it was exposed by accident in 4.1) :grimacing: 
Fixed

Reply:
If it's exposed by accident, I would argue we should delete it for all versions, to avoid forward compatibility issues. So you could move it down to the general `match` statement.